scripts/osx/testTemplate.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+OF_DIR=`pwd`;
+echo $OF_DIR;
+TEST_DIR="$OF_DIR/scripts/osx"
+cd $TEST_DIR
```

Comment:
> script would work independently of travis

ah, good point. however, afaict, this script only works when executed from the OF root dir, not from the scripts directory, due to the pwd..